index.js

```diff
@@ -143,6 +144,12 @@ function RingPop(options) {
 
     this.destroyed = false;
     this.joiner = null;
+
+    this.startTime = Date.now(); //used for calculating uptime
+
+    var dependencies = packageJSON.devDependencies;
+    this.tchannelVersion = dependencies.tchannel;
```

Comment:
We'll need to get the tchannel version in another way. The dev dependency won't suffice since, in production, the dev version won't be the one that is running. You can get the tchannel version like this:

```
require(path.join(path.dirname(require.resolve('tchannel')), 'package.json')).version
```